cds_dojson/schemas/records/project-v1.0.0.json

```diff
@@ -191,6 +191,9 @@
               "type": "integer"
             }
           ]
+        },
+        "state": {
+          "type": "object"
```

Comment:
Do we need this guy here, I mean, once the deposit is published we don't need to know the state anymore (actually everything should be done)